src/translib/common_app.go

```diff
@@ -235,9 +235,9 @@ func (app *CommonApp) translateCRUDCommon(d *db.DB, opcode int) ([]db.WatchKeys,
 	OrdTblList, err = transformer.GetOrdDBTblList(moduleNm)
 	if (err != nil) || (len(OrdTblList) == 0) {
 		log.Error("GetOrdDBTblList() failed")
-		return keys, err
+		//return keys, err
 	}
-
+        OrdTblList = []string {"FDB_TABLE"}
```

Comment:
I dont think, this can be commited into project-arlo master, please ask infra team to commit the infra changes.

Reply:
removed changes from my branch.. The infra team will address this issue.